dev/io.openliberty.org.apache.myfaces.4.1_fat/build.gradle

```diff
@@ -20,11 +20,13 @@
 
 // Define G:A:V coordinates of each dependency
 dependencies {
+    requiredLibs 'org.seleniumhq.selenium:selenium-java:4.8.3'
+    requiredLibs 'com.google.guava:guava:32.0.1-jre'
     requiredLibs 'net.sourceforge.htmlunit:htmlunit:2.44.0'
     requiredLibs 'net.sourceforge.htmlunit:htmlunit-cssparser:1.6.0'
     requiredLibs 'net.sourceforge.htmlunit:htmlunit-core-js:2.44.0'
     requiredLibs 'net.sourceforge.htmlunit:neko-htmlunit:2.44.0' 
-    requiredLibs 'org.apache.commons:commons-lang3:3.14.0'
+    requiredLibs 'org.apache.commons:commons-lang3:3.8'
```

Comment:
Why the downgrade here?

Reply:
Reverted the change -- not sure what happened